### PR TITLE
Update podfile

### PR DESCRIPTION
**Pull request by alanscarpa**
**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ source 'https://github.com/CocoaPods/Specs'
 platform :ios, '9.0'
 
 target 'TARGET_NAME' do
-    pod 'TwilioVideo', '~> 2.0'
+    pod 'TwilioVideo', '~> 3.0.0'
```
Comment (ceaglest) [inline README.md]: Would it be possible to allow any minor version? Twilio Video uses semVer.

```suggestion
    pod 'TwilioVideo', '~> 3.0'
```

**Review by ceaglest — CHANGES_REQUESTED** [review]
Hi @alanscarpa thank you for submitting a PR to fix this inconsistency in our docs! 

I left a comment about the podfile. If you could address and check "I acknowledge that all my contributions will be made under the project's license." then I would be happy to accept the change.

File: README.md (hunk 1 above)
Comment (alanscarpa) [reply README.md]: Ah, good point!

**Comment by alanscarpa** [comment]
I clicked the checkbox. :) Thanks for all the good work.

**Review by ceaglest — APPROVED** [review]

**Pull request merged**